tests/unit/misc/test_sessions.py

```diff
@@ -32,7 +32,11 @@
 from qutebrowser.browser.webkit import tabhistory
 
 
-pytestmark = pytest.mark.qt_log_ignore('QIODevice::read.*: device not open')
+@pytest.fixture(autouse=True)
+@pytest.mark.qt_log_ignore('QIODevice::read.*: device not open')
+def autouse(fake_save_manager):
+    yield
```

Comment:
You can also use `pass` instead, which seems nicer to me than yielding into the void :wink: